specification/monitor/resource-manager/Microsoft.Monitor/stable/2023-04-03/examples/AzureMonitorWorkspacesDelete.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters": {
+    "subscriptionId": "703362b3-f278-4e4b-9179-c76eaf41ffc2",
+    "resourceGroupName": "myResourceGroup",
+    "azureMonitorWorkspaceName": "myAzureMonitorWorkspace",
+    "api-version": "2023-04-03"
+  },
+  "responses": {
+    "200": {},
```

Comment:
The API is changing to asynchronous soon. 200 should be replaced with 202.

Reply:
Updated